### ISSUE TITLE
Unable to "Click" in Certain Menus

### ISSUE THREAD

**Issue by garagevive** [Eusth/HoneySelectVR#35]
I am able to click with the vive controller in about half the menus. However, many important menus do not allow me to click in them and I have to use the mouse. IE: I can select a category of actions but cannot click the action itself without using my physical mouse.

Is this a known issue?

**Comment by Eusth** [comment]
Works for me (if you are talking about the menu with the positions). Make sure that you are using the latest version and the game window isn't covered by anything.

**Comment by zebitty** [comment]
Yes, I found that if I use the thumbstick on the touch controller to navigate around and then push the thumbstick to click, that always seemed to work, but using the laser pointer to navigate I got the same problem as op. Mostly when trying to select positions.

**Comment by Nordomus** [comment]
I have the same issue with Vive controllers :( And how can I change resolution/window mode? Can't find those options ingame. Oh and I also don't know how to initialize moves and control them. It was easy in Play Club, here I have no idea, maybe something is not working as well?

**Comment by Eusth** [comment]
InitSetting.exe

**Comment by sycspysycspy** [comment]
I am a oculus rift owner, and the click is on the trigger, I found it hard to do click as when I pull the trigger most of the time it recognise as click and hold rather than click.

**Comment by iasdiess** [comment]
i reck i read somewhere you need to insure the app on your screen computer remains on top for the select/click to be recognized; i used to have the same issue, now no longer (oculus)
